docs/fieldnorm/user_feedback.md

```diff
@@ -0,0 +1,21 @@
+# Laboratory patients, user, and personnel feedback
+
+Bioinformaticians develop data analysis workflows, tools and (web) applications that are used by different groups of laboratory analists and specialists, from hereafter refered as 'end users'. These end users will experience events where unexpected, incorrect or occasionally no results are produced, due to bugs which are caused by either errors, flaws or faults in the design, development, or operation of the software developed by the bioinformaticians. In addition, end users may come up with requests and suggestions for new features / functionality to further improve or expand the scope of existing software. 
+
+The bioinformatics team therefore needs to setup a system through which both reports of issues / bugs and requests can be submitted, traced and processed. Rather than using a single procedure / system to handle both issues and new requests or feedback, different procedures may be set up to process and handle them independently in a different way. When the issues have been handled and actions have been taken, results need to be communicated back to the end users. 
+
+The requirements of the system are :
+
+-   Feedback / requests from end users can be submitted, registered and traced by the system. There is a documented procedure for end users on how to use the system to submit feedback / requests and how they will be processed. 
+-   Feedback / requests are evaluated and an evaluation is made as to which actions are needed and which requirements need to be met.  
+-   Actions that are being taken to process the feedback / requests are recorded and can be traced back to it either by the system itself or by reference to other records and documentation. 
+-   Results are communicated back to the end users with reference to the initial feedback / request provided. 
+
+Examples of systems that may be used (together) to collect and register feedback and requests that can be configured to meet the stated requirements are
+
+- A special purpose (non-Personal) email address 
+- A ticketing system 
+- A web / service portal where issues / requests can be submitted by filling out forms
+- Lab meetings where feature requests are discussed and minutes are recorded and documented
+- Built-in issue trackers of (git) version control systems such as Github or Gitlab
+- Work management / Task planner systems such as JIRA, Microsoft Teams Planner
```

Comment:
The 2022 version of the ISO is more risk based. And depending on the risks for example; more audits should be done.
As most of the bugs/issues are only recorded with the bioinformaticians, they never end up in the "managementevaluation" statistics; hence there will no actions been taken for bioinformatics.
We should think about how to incorporate the issues/problems reported to bioinformaticians, and how to get this back into the QMS/management evalation things of the lab. 
I have no clear answer about this one yet ;)

Reply:
we should somehow refer to "nonconfirming work", as bugs are part of this section of the ISO
we do deal with this in a similar way.    
Or we make it a new 'section' and refer to similar procedures as the feedback. And then add a bit about how to incorporate this into QMS of the lab....

Reply:
This will always be tough. When is a bug a non-conformity. Is this always the case? or only if it has an impact on diagnostics? and how do you determine when a bug "impacts" diagnostics or not. And if so, how are they handled? how are they tracked? who evaluates them? 

At our department we sort of try to determine when a bug has a high impact on the diagnostic process and should be handled through the departments way of handling nonconformities, but this is mostly nattevingerwerk.